### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'toLowerCase')     at devtools-overlay.mjs:8:183129

### ISSUE THREAD

**Issue by mosessmax** [vuejs/devtools-next#516]
i get this error, as soon as i type into the input field, it goes away on refresh but shows up again as soon as i type in the input.

version: "vite-plugin-vue-devtools": "^7.3.1",

![CleanShot 2024-07-10 at 14 51 33](https://github.com/vuejs/devtools-next/assets/40890902/d6f537be-8588-43cf-b8dc-d79b1f0aee55)

**Comment by webfansplz** [comment]
Can you provide a mini repo? Thanks

**Comment by r4g309** [comment]
I get the same error. I think it has to do with the autocomplete option. When I click and it fills out the form, that's when the error occurs.

```vue
<script setup lang="ts"></script>

<template>
  <input name="firstName" type="text" autocomplete="given-name" />
</template>
```
![image](https://github.com/user-attachments/assets/08296b74-30f8-4a7a-8df9-580802aed1e2)
![image](https://github.com/user-attachments/assets/9da6adc5-aedf-470b-a7da-542f23f73e7a)

**Comment by phox081** [comment]
I get the same error by clicking anywhere on the interface, but only on the first click. This happens in a Vue.js project that was created before adding the plugin. In a clean Vue.js project, I don't encounter the error.

**Comment by mosessmax** [comment]
> I get the same error by clicking anywhere on the interface, but only on the first click. This happens in a Vue.js project that was created before adding the plugin. In a clean Vue.js project, I don't encounter the error.

it was a clean vue.js project, i selected the devtool plugin while scaffolding the project.

**Comment by webfansplz** [comment]
> > I get the same error by clicking anywhere on the interface, but only on the first click. This happens in a Vue.js project that was created before adding the plugin. In a clean Vue.js project, I don't encounter the error.
> 
> it was a clean vue.js project, i selected the devtool plugin while scaffolding the project.

Can you share the repo to help us track it? Thanks.

**Comment by webfansplz** [comment]
I cannot reproduce it, so if someone can provide a mini repo, it would be helpful.

**Comment by Deckluhm** [comment]
> I cannot reproduce it, so if someone can provide a mini repo, it would be helpful.

Here is a reproduction: https://github.com/Deckluhm/vue-devtools-next

⚠️ Important to note that it seems to only happen on Chromium-based browsers, not on Firefox

**Comment by Deckluhm** [comment]
Also, I forgot to mention that, as @r4g309 said, it seems to only happen when using an autocomplete suggestion.

**Comment by webfansplz** [comment]
Fixed in #502 , It will work fine in the next patch release
